transport/src/main/java/io/netty5/channel/DefaultChannelId.java

```diff
@@ -183,6 +194,7 @@ private String newLongValue() {
         longToHexStringPadded(buf, mixedTimeStampId);
         buf.append('-');
         intToHexStringPadded(buf, randomId);
+        assert buf.capacity() == buf.length();
```

Comment:
Why this assert?

Reply:
My intention was to ensure that no doubling occurs while constructing the `newLongValue()`. However, I can now see that it isn't working properly. I'll remove it.